sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -677,14 +683,22 @@ func getSloBurnRateTimeRangeSchema(triggerType string) *schema.Schema {
 	return &schema.Schema{
 		Type:             schema.TypeString,
 		Optional:         true,
-		ValidateFunc:     timeRangeValidation,
+		ValidateFunc:     timeRangeFormatValidation,
 		DiffSuppressFunc: SuppressEquivalentTimeDiff(false),
 		RequiredWith:     []string{requiredWith},
 		ConflictsWith:    []string{conflictsWith},
 	}
 }
 
-var timeRangeValidation = validation.StringMatch(regexp.MustCompile(`^-?(\d)+[smhd]$`), "Time range must be in the format '-?\\d+[smhd]'. Examples: -15m, 1d, etc.")
+var allowedTimeRanges = []string{
+	"5m", "10m", "15m", "30m", "60m", "180m", "360m", "720m", "1440m",
+	"-5m", "-10m", "-15m", "-30m", "-60m", "-180m", "-360m", "-720m", "-1440m",
+	"1h", "3h", "6h", "12h", "24h",
+	"-1h", "-3h", "-6h", "-12h", "-24h",
+	"-1d", "1d",
```

Comment:
We allow 2m time range for some customers. Please see `logs.monitor.allowed.time.durations` and `metrics.monitor.allowed.time.durations` properties. That's the reason we don't have a validation for this in Terraform and OpenAPI schema. I see two options:
- Leave it as is.
- Allow 2m and -2m here as well.
#2 is not too bad since if 2m time range is not allowed for a customer, it will fail at run time for them.

Reply:
Ok, I've added 2m and -2m